subscriptions/templatetags/seed.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+import seed_stage_based_messaging
+
+register = template.Library()
+
+
+@register.simple_tag
+def current_version():
+    return seed_stage_based_messaging.__version__
```

Comment:
Having the version in the main package init file seems to be a convention used by Django and DRF but I am open to changing it if a VERSION file is preferred?